diff --git a/dist/index.mjs b/dist/index.mjs
index fce743da07ec6daa688d86e516b2a90e8610a27f..a40f4f929178c05874561ee34974a053d73d0255 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -4105,7 +4105,7 @@ function findRegisteredRedirectUri(registered, requested) {
 		if (!req) return false;
 		try {
 			const reg = new URL(url);
-			return isLoopbackIP(reg.hostname) && reg.hostname === req.hostname && reg.pathname === req.pathname && reg.protocol === req.protocol && reg.search === req.search;
+			return isLoopbackHost(reg.hostname) && reg.hostname === req.hostname && reg.pathname === req.pathname && reg.protocol === req.protocol && reg.search === req.search;
 		} catch {
 			return false;
 		}
